v3data/config.py

```diff
@@ -148,7 +148,7 @@
             os.environ.get("GAMMA_SUBGRAPH_MAINNET", "prod")
         ],
         "polygon": gamma_subgraphs["uniswap_v3"]["polygon"][
-            os.environ.get("GAMMA_SUBGRAPH_POLYGON", "prod")
+            os.environ.get("GAMMA_SUBGRAPH_POLYGON", "test")
```

Comment:
Keep default as prod and set this in env var.

Reply:
my bad.. didnt want that .. ok.